06-files-input-and-output/csv_split.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2016 Ami . All rights reserved
+import argparse
+import csv
+import os
+import sys
+
+def get_arguments():
+    current_directory = os.getcwd()
+    parser = argparse.ArgumentParser(description="Split CSV by every x rows")
+
+    parser.add_argument(
+        "-i",
+        action="store",
+        dest="input_file",
+        help="CSV input file",
+        required=True
+    )
+    parser.add_argument(
+        "-o",
+        action="store",
+        dest="output_path",
+        help="Output path - where to store the split CSV files",
+        default=current_directory
+    )
+    parser.add_argument(
+        "-r",
+        action="store",
+        dest="row_limit",
+        help="split by every x lines",
+        type=int,
+        required=True
+    )
+
+    if len(sys.argv) == 1:
+        parser.print_help()
+        return False
```

Comment:
Return the equivalent of False but in using the type that parser.parse_args() returns. For example, if parse_args returns a list, return an empty list.